services/condor-log-monitor/Dockerfile

```diff
@@ -9,9 +9,6 @@ LABEL org.iplantc.de.condor-log-monitor.git-ref="$git_commit" \
 
 RUN useradd -m -U -s /bin/bash iplant
```

Comment:
This is part of the same process. But I also think clm might just be in the jex-events image now, so... not sure if it matters? @johnworth would know I guess